tests/tf/models/test_retrieval.py

```diff
@@ -46,21 +46,28 @@ def test_two_tower_retrieval_model_with_metrics(ecommerce_data: SyntheticData, r
         loss="categorical_crossentropy",
     )
 
+    # Setting up evaluation
+    item_features = ecommerce_data.schema.select_by_tag(Tags.ITEM).column_names
+    item_dataset = ecommerce_data.dataframe[item_features].drop_duplicates()
+    item_dataset = item_dataset.sort_values("item_id")
```

Comment:
Do we need to sort by "item_id"? If that is required, can maybe do that inside `model.set_evaluation_candidates()` ?

Reply:
We don't need the sorting step, I've just added it for some internal tests but forgot to remove it.